lib/route.js

```diff
@@ -285,11 +302,9 @@ function route(name, model, schema, routeOptions) {
                         types.push(key);
                         linkedResources = linkedResources.map(function (resource) {
                             // find out which resources are linked to by this collection
-                            var associations = _.filter(
-                                getAssociations.call(_this, _this._schema[singularKey]),
-                                function (association) {
-                                    return association.type === collection;
-                                });
+                            var associations = _.filter(getAssociations(harvester._schema[singularKey], harvester.options), function (association) {
```

Comment:
We actually prefer when long calls are broken up and spread across lines - it makes the code way more readable than a single line would be.